### ISSUE TITLE
Change summoner web interface to have full addresses

### ISSUE THREAD

**Issue by hdevalence** [penumbra-zone/penumbra#3626]
**Is your feature request related to a problem? Please describe.**

Many users are confused when they check the summoning page and see a different address. If the address was shown, they could copy it and do pcli v address to check that it was theirs. But only the short form is shown, so they can't.

**Describe the solution you'd like**

Change the column ordering to

Number
Date
Hash
Full Address

**Comment by conorsch** [comment]
Isn't the point of the contribution hash to that folks can copy/paste a string and search for it? Optically, it makes sense to me that the summoner page defaults to short addresses, because it's consistent with a privacy-first mentality. If the pcli ceremony uses a discrete address other than index 0, maybe we can clarify that info in user-facing docs.

**Comment by hdevalence** [comment]
> Isn't the point of the contribution hash to that folks can copy/paste a string and search for it?

Yep, but sometimes people don't, try to look up their contribution by slot number, and then get confused because the address doesn't match. Because we only display their contribution address at the _start_ of their ceremony contribution, they don't have it on hand. And because we don't display the _full_ address, we can't tell them to plug it into `pcli view address` to confirm that it's theirs.

> Optically, it makes sense to me that the summoner page defaults to short addresses, because it's consistent with a privacy-first mentality.

The addresses are in the transcript though, so they are public -- hiding them in the web UI doesn't actually protect them. So I think we might as well publish them on the site.

**Comment by conorsch** [comment]
> The addresses are in the transcript though, so they are public -- hiding them in the web UI doesn't actually protect them. 

Great point, thanks for clarifying.

**Comment by cronokirby** [comment]
Another mitigating factor is that you can't contribute with the same address twice anyways (summonerd disallows this), and we use a different address for each phase. So, after you've already contributed, the secrecy of that address doesn't matter anymore.
